assets/js/app.js

```diff
@@ -157,4 +157,12 @@ $(function(){
 
 });
 
+//Navbar current page highlight
```

Comment:
For projects of this size, I would suggest using multi-line comments for code-blocks

```
/*
-------------------------------------
    NAV BAR HIGHTLIGHTING
--------------------------------------
*/
```

This is helpful in the log term

Reply:
@jainh6148 Could you please add the comments as requested 😅